content/en/blog/_posts/2024-05-13-Gateway-API-v1.1/index.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: blog
+title: "Gateway API v1.1: Feature Graduation and Other Updates"
+date: 2024-05-13T19:00:00-08:00
+slug: gateway-api-v1-1
+---
+
+![Gateway API Logo](gateway-api-logo.png)
+**Authors:** 
+GRPCRoute - Richard Belleville (Google), 
+ParentRef Port - Frank Budinsky (IBM),
+Gateway Client Certificate Validation - Arko Dasgupta (?),
+GAMMA - John Howard (Google),
+Conformance Profiles and Reports - Mattia Lavacca (Kong), Christine Kim (?),
+Session Persistence and BackendLBPolicy - Grant Spence (Red Hat)
+TLS Terminology Clarifications - Candace Holman (Red Hat),
+and other reviewer and release note contributors.
+
+In the wake of the GA release of Gateway API last October, the Kubernetes
+SIG Network buoyantly announces the v1.1 release
+of [Gateway API](https://gateway-api.sigs.k8s.io/). Gateway API features for
+**GRPCRoute**, **ParentRef Port**, and **Conformance Profiles and Reports** will
+graduate to
+Standard. In addition, **Gateway Client Certificate Verification** and **Session
+Persistence** are adopted as new, Experimental features. **Gateway API for
+Service Mesh (GAMMA)** support, and clarifications to TLS
+terminology and gateway merging are added to strengthen the release.
```

Comment:
```suggestion
**Gateway API for Service Mesh (GAMMA)**, **GRPCRoute**, **ParentRef Port**, 
and **Conformance Profiles and Reports** will graduate to Standard.
In addition, **Gateway Client Certificate Verification** and **Session
Persistence** are adopted as new, Experimental features. Clarifications to TLS
terminology and gateway merging are added to strengthen the release.
```

Updated to move GAMMA up to Standard section to reflect grouping/ordering below.